rmgpy/species.py

```diff
@@ -327,6 +327,10 @@ def is_identical(self, other, strict=True):
             for molecule in self.molecule:
                 if molecule.is_identical(other, strict=strict):
                     return True
+        elif isinstance(other, Fragment):
+            for molecule in self.molecule:
+                if molecule.is_identical(other, strict=strict):
+                    return True
```

Comment:
Can this be combined with the block above it since the code for Molecule objects is the exact same? (e.g. `if isinstance(other, Molecule) or isinstance(other, Fragment): ...` )

Reply:
Done!